subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap.c

```diff
@@ -212,6 +212,24 @@ int nrf_cloud_coap_pgps_url_get(struct nrf_cloud_rest_pgps_request const *const
 }
 #endif /* CONFIG_NRF_CLOUD_PGPS */
 
+
+int nrf_cloud_coap_bytes_send(uint8_t *buf, size_t buf_len)
+{
+	int err = 0;
+
+	if (!nrf_cloud_coap_is_connected()) {
+		return -EACCES;
+	}
+
+	err = nrf_cloud_coap_post("/msg/d2c/raw", NULL, buf, buf_len,
```

Comment:
can you add `#define COAP_D2C_RAW_RSC "/msg/d2c/raw"` just to be consistent with the other resource paths?